circuit_knitting/cutting/qpd/instructions/cut_wire.py

```diff
@@ -0,0 +1,24 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Single-qubit instruction to denote a wire cut location."""
+from __future__ import annotations
+
+from qiskit.circuit import Instruction
+
+
+class CutWire(Instruction):
```

Comment:
It _might_ be more appropriate for this to derive from `Gate`, since a `CutWire` represents an identity gate, which is unitary.  Also, it might make sense for it to have a `_define` method, just like `Move` does.